src/cgame/cg_ents.cpp

```diff
@@ -28,6 +28,12 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
 #include "cg_local.h"
 
+/* It is dynamically adjusted in CG_AddPacketEntities() to avoid looping
+over unused entities. It can be zero before the first frame starts adding
+entities, as none is added yet and none has to be removed yet. This should
+never reach the MAX_GENTITIES value. */
+int cg_highestPreviousEntityNumberInPVS = 0;
```

Comment:
The PVS wording should be removed since it includes all entities known to the cgame including beacons which are not in the PVS. I think "previous" is not good either since it's current for everywhere except CG_AddPacketEntities.

Maybe `cg_highestActiveEntity`

Reply:
Done.